examples/maps/maps.description

```diff
@@ -1,5 +1,5 @@
-// The `map<T>` type is a `map` from strings to `T`. `map` syntax is similar to JSON. Maps are mutable: `m[k]` is an
-// `lvalue`. `foreach` will iterate over the values of the `map`. `m[k]` gets entry for `k`; `nil` if missing.
-// Use `m.get(k)` when you know that there is an entry for `k`. `m.keys()` can be used to iterate over keys,
-// to get the keys as an `array` of strings. `==` and `!=` on maps is deep: two maps are equal if they
-// have the same set of keys and the values for each key are equal.
+// The `map<T>` type is a `map` from strings to `T`. The `map` syntax is similar to JSON. Maps are mutable.
+// `m[k]` is an `lvalue`. `foreach` will iterate over the values of the `map`. `m[k]` gets entry for `k`; `nil` if
```

Comment:
```suggestion
// `m[k]` is an lvalue. `foreach` will iterate over the values of the `map`. `m[k]` gets entry for `k`; `nil` if
```
Is there a need to emphasize "lvalue"? 🤔 If so, how about using quotes for that instead of formatting it as code?